keras/layers/convolutional.py

```diff
@@ -1423,22 +1431,29 @@ class ZeroPadding1D(Layer):
         3D tensor with shape (samples, padded_axis, features)
     '''
 
-    def __init__(self, padding=1, **kwargs):
+    def __init__(self, padding=1, left_pad=1, right_pad=1, **kwargs):
```

Comment:
Same here, I think we should have a polymorphic padding argument that could be `int`, tuple of size 2, or dictionary.